sds/src/scanner/mod.rs

```diff
@@ -832,6 +855,95 @@ mod test {
         assert_eq!(matches.len(), 1);
     }
 
+    #[test]
+    fn test_included_keywords_match_path() {
+        let redact_test_rule = RegexRuleConfig::builder("world".to_owned())
+            .match_action(MatchAction::Redact {
+                replacement: "[REDACTED]".to_string(),
+            })
+            .proximity_keywords(ProximityKeywordsConfig {
+                look_ahead_character_count: 30,
+                included_keywords: vec!["awsAccess".to_string(), "access/key".to_string()],
+                excluded_keywords: vec![],
+            })
+            .build();
+
+        let scanner = Scanner::new(&[redact_test_rule]).unwrap();
```

Comment:
As the `should_keywords_match_event_paths` parameter is pretty important here, I think the test should make it explicit that `should_keywords_match_event_paths` is expected to be true in this test. I think it would be interesting to have also a test where `should_keywords_match_event_paths` is false.

Reply:
Yes I agree 👍

Reply:
I did it in [Split tests and move path sanitization](https://github.com/DataDog/dd-sensitive-data-scanner/pull/53/commits/4c3068390ed5fce0bcadc0f6471fa1586bda910e)